docs/vaccel_go.md

```diff
@@ -26,101 +26,56 @@ about Go installation can be found [here](https://go.dev/doc/install),
 but there are also instructions on how to install `Go` in the
 [vAccel-go bindings installation guide](https://github.com/nubificus/go-vaccel).
 
-### vAccel Go package
+## vAccel Go package
 The vaccel package in Golang provides access to vAccel operations, which
 can be used by the developers on their own `Go` programs. The vaccel package uses 
-the native `C` bindings in order to use the vAccel `C` API. The following diagram 
-demonstrates the functionality of the vaccel package:
+the native `C` bindings in order to use the vAccel `C` API.
```

Comment:
I'd say ditch everything up to this point and rephrase this to something like:
[Go](https://go.dev/) bindings for vAccel are implemented in the `vaccel` Go package. The `vaccel` package leverages the vAccel C API to provide native Go support for vAccel operations.